inference-engine/tests/unit/gna/ngraph/transformations/gna_broadcast_const.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "transformations/broadcast_const.hpp"
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <transformations/init_node_info.hpp>
+
+namespace testing {
+
+std::shared_ptr<ngraph::opset8::FakeQuantize> createFakeQuantizeNode(std::shared_ptr<ngraph::op::Op> parent_node) {
+    auto input_low = ngraph::opset8::Constant::create(ngraph::element::f32, {}, {-0.5});
+    auto input_high = ngraph::opset8::Constant::create(ngraph::element::f32, {}, {0.5});
+    auto output_low = ngraph::opset8::Constant::create(ngraph::element::f32, {}, {-0.5});
+    auto output_high = ngraph::opset8::Constant::create(ngraph::element::f32, {}, {0.5});
+    return std::make_shared<ngraph::opset8::FakeQuantize>(parent_node, input_low,
+                                                          input_high, output_low,
+                                                          output_high, 0);
+}
+
+TEST(TransformationTests, BroadcastConstTestFakeQuantize) {
+    std::shared_ptr<ngraph::Function> func(nullptr), reference_func(nullptr);
+    const ngraph::Shape data_shape{3, 2};
+
+    auto create_graph = [](const ngraph::Shape& data_shape, const ngraph::Shape& const_shape_dims,
+                           const ngraph::Shape& const_shape_values) {
+        auto input_params = std::make_shared<ngraph::opset8::Parameter>(ngraph::element::f32, data_shape);
+
+        auto constant = ngraph::opset8::Constant::create(ngraph::element::Type_t::f32,
+                                                         ngraph::Shape{const_shape_dims}, const_shape_values);
+
+        auto fakeQuantize1 = createFakeQuantizeNode(input_params);
+        auto fakeQuantize2 = createFakeQuantizeNode(constant);
+
+        auto add = std::make_shared<ngraph::opset8::Add>(fakeQuantize1, fakeQuantize2);
```

Comment:
I'd also check the opposite case:
 auto add = std::make_shared<ngraph::opset8::Add>(fakeQuantize2, fakeQuantize1);